other/bootstrap_daemon/src/tox-bootstrapd.c

```diff
@@ -366,6 +382,7 @@ int main(int argc, char *argv[])
             free(motd);
         } else {
             log_write(LOG_LEVEL_ERROR, "Couldn't set MOTD: %s. Exiting.\n", motd);
+            kill_gca(group_announce);
             kill_onion_announce(onion_a);
```

Comment:
Before your change, the destruction order was the reverse of the construction order, as if the stack was unwinding. However, you seem to insist on destroying `group_announce` first in all of your `tox-bootstrapd.c` changes. Is there a reason for this?

```suggestion
            kill_onion_announce(onion_a);
            kill_gca(group_announce);
```